RetroBar/Languages/polski.xaml

```diff
@@ -53,30 +59,48 @@
     <s:String x:Key="start_text">Start</s:String>
     <s:String x:Key="start_text_xp">Start</s:String>
     <s:String x:Key="start_button_tip_98">Kliknij tutaj aby rozpocząć.</s:String>
-    <s:String x:Key="start_button_tip">Kliknij, aby rozpocząć</s:String>
+    <s:String x:Key="start_button_tip">Kliknij tutaj aby rozpocząć</s:String>
```

Comment:
![XPPol](https://github.com/dremin/RetroBar/assets/28744392/6399ffc9-8281-4664-8c34-2e9d472a82ad)
The previous translation seems to be more accurate, why did you change it?